static/js/integrations_dev_panel.js

```diff
@@ -0,0 +1,173 @@
+}
+
+function load_fixture_body(fixture_name) {
+    var integration_name = get_selected_integration_name();
+    var element = fixtures_collection[integration_name][fixture_name];
+    var fixture_body = JSON.stringify(element, null, 4);
+    if (fixture_body === undefined) {
+        alert("Fixture does not have a body.");
+    }
+    $("#fixture_body")[0].value = fixture_body;
+}
+
+function handle_unsuccessful_response(response) {
+    try {
+        response = JSON.parse(response.responseText);
+        set_message(response.msg, "warning");
+    }
+    catch(err) {
+        console.log(err);  // A bit paranoid, but why not?
+        document.write(response.responseText);
+    }
+    clear_fixture_body();
+    clear_fixture_name();
+    clear_integration_name();
+    return;
+}
+
+function get_fixtures(integration_name) {
+    if (integration_name === "") {
+        clear_fixture_body();
+        clear_fixture_name();
+    }
+    if (fixtures_collection[integration_name] === undefined) {
+        // final relative url pattern: devtools/integrations/(?P<integration_name>.+)/fixtures
+        $.ajax({
+            url: integration_name + "/fixtures",
+            type: "GET",
+            success: function (response) {
+                fixtures_collection[integration_name] = response.fixtures;
+                fixtures_list = $("#fixture_name")[0];
+                var fixtures_names = Object.keys(response.fixtures)
+                fixtures_names.forEach( function(fixture_name) {
+                        var new_option = document.createElement("option");
+                        new_option.value = fixture_name;
+                        new_option.innerHTML = fixture_name;
+                        fixtures_list.add(new_option);
+                    }
+                );
+                load_fixture_body(fixtures_names[0]);
+                return;
+            },
+            error: handle_unsuccessful_response,
+        });
+    }
+}
+
+function send_webhook_fixture_message() {
+    var integration_name = get_selected_integration_name();
+    if (integration_name === "") {
+        set_message("Must select an integration.", "warning");
+        return;
+    }
+
+    var payload = $("#fixture_body")[0].value;
+    try {
+        // Perform a quick check to make sure that we are sending valid JSON.
+        payload = JSON.parse(payload);
+    }
+    catch (err) {
+        set_message("Inavlid JSON.", "warning");
+        return;
+    }
+    // final relative url pattern: devtools/integrations/
+    var csrftoken = $("#csrftoken").val();
+    $.ajax({
+        url: "",
+        type: "POST",
+        data: payload,
+        beforeSend: function(xhr){xhr.setRequestHeader('X-CSRFToken', csrftoken);},
+        success: function (response) {
+            set_message(response.msg, "success");
+            return;
+        },
+        error: function (response) {
+            set_message("UnSuccessful!", "warning");
```

Comment:
Replace `UnSuccessful` with `Unsuccessful!` :)

Reply:
That was a temporary placeholder. I'll be replacing it.